### ISSUE TITLE
Running Sffio.py fails with following error.

### ISSUE THREAD

**Issue by vincentdavis** [biopython/biopython#946]
Could someone confirm this? I am working on this in reference to issue #820 
```python
Traceback (most recent call last):
  File "Sffio.py", line 1386, in <module>
    for record in SffIterator(handle):
  File "Sffio.py", line 898, in SffIterator
    assert 0 == handle.tell(), "Not at start of file, offset %i" % handle.tell()
AssertionError: Not at start of file, offset 65296

```

**Comment by peterjc** [comment]
Some of the tests do fun stuff with the handle position, I'd need more context. Point here is ``SffIterator`` expects the handle to be at the start of the file (offset zero), and it was not.

**Comment by vincentdavis** [comment]
@peterjc I am just running the file directly.
```bash
$ python SffIO.py
Running quick self test
E3MFGYR02JWQ7T
E3MFGYR02JA6IL
E3MFGYR02JHD4H
E3MFGYR02GFKUC
E3MFGYR02FTGED
E3MFGYR02FR9G7
E3MFGYR02GAZMS
E3MFGYR02HHZ8O
E3MFGYR02GPGB1
E3MFGYR02F7Z7G
Writing with a list of SeqRecords...
And again with an iterator...
--------------------------------------------------
alpha
beta
gamma
delta
epsilon
zeta
eta
theta
iota
kappa
lambda
mu
nu
xi
omicron
pi
rho
sigma
tau
upsilon
phi
chi
psi
omega
Traceback (most recent call last):
  File "SffIO.py", line 1386, in <module>
    for record in SffIterator(handle):
  File "SffIO.py", line 898, in SffIterator
    assert 0 == handle.tell(), "Not at start of file, offset %i" % handle.tell()
AssertionError: Not at start of file, offset 65296
```

**Comment by peterjc** [comment]
Sorry, I got the same. Looks like bit-rot, should be fixed with 857c8913ea09ac37a59eead088183ae12fed2947

Thanks for reporting this.
